ui.apps/src/main/content/jcr_root/apps/venia/clientlibs/common/js/CommerceGraphqlApi.js

```diff
@@ -0,0 +1,94 @@
+/*******************************************************************************
+ *
+ *     Copyright 2019 Adobe. All rights reserved.
+ *     This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License. You may obtain a copy
+ *     of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software distributed under
+ *     the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ *     OF ANY KIND, either express or implied. See the License for the specific language
+ *     governing permissions and limitations under the License.
+ *
+ ******************************************************************************/
+
+
+(function () {
+
+    'use strict';
+    //todo update the proxy rules so that this gets router through the dispatcher
```

Comment:
Should be done in same PR?

Reply:
@LSantha @cjelger I've updated the proxy configuration to include the URLs to the product images.